pkg/netpol/connlist/exposed_pods.go

```diff
@@ -0,0 +1,16 @@
+package connlist
+
+import "github.com/np-guard/netpol-analyzer/pkg/netpol/internal/common"
+
+// ExposedPeer captures potential ingress and egress connections data for an exposed Peer
+type ExposedPeer interface {
+	ExposedPeer() Peer // UnprotectedPeer()
+	IngressExposure() []XgressExposureData
+	EgressExposure() []XgressExposureData
+}
+
+// XgressExposureData data of potential connectivity for an existing peer to/from a representative peer
+type XgressExposureData interface {
+	RepresentativePeer() Peer
```

Comment:
`RepresentativePeer` is an internal implementation detail, and not something to expose..  I'd like to expose instead the actual relevant details: exposed namespace by labels / exposed workloads by labels / is exposed to entire cluster  (all namespaces) / is not protected by any netpol (per each direction) ? 

For the implementation of course, the `RepresentativePeer` will have to implement this api.

Reply:
I am planning to implement `RepresentativePeer` in next PR
in this PR to focus on the implementations should be done to `connlist` and use the data I already have from the first (base of this) branch , do you agree?